### PR TITLE
Redirect awslogs agent logs to a file

### PR DESCRIPTION
**Pull request by allait**
awslogs agent is quite noisy, especially when running the container
locally without valid AWS credentials.

Since most of the time we're not interested in the messages generated
by awslogs agent but we might need them to investigate any potential
issues we can redirect the agent output to a 5MB file with supervisor.

This file is not itself shipped to CloudWatch, since any problems with
the agent we'd be interested in investigating would probably prevent it
from shipping logs to CloudWatch in the first place.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.0.2
+1.0.3
```
Comment (Wynndow) [inline VERSION]: Should this be 1.1.0?

File: supervisord.conf [hunk 2]
```diff
@@ -30,6 +30,7 @@ autostart = true
 autorestart = true
 stdout_logfile = /dev/stdout
 stdout_logfile_maxbytes = 0
-stderr_logfile = /dev/stderr
-stderr_logfile_maxbytes = 0
+stderr_logfile = /var/log/awslogs.log
+stderr_logfile_maxbytes = 5000000
+stdout_logfile_backups = 3
```
Comment (Wynndow) [inline supervisord.conf]: `stderr_logfile_backups` here I think

File: VERSION (hunk 1 above)
Comment (allait) [inline VERSION]: It's not really adding anything new, just tweaking some settings. I'd say it's very similar to renaming the nginx log group

File: supervisord.conf (hunk 2 above)
Comment (allait) [inline supervisord.conf]: Oops! Nice catch.

File: VERSION (hunk 1 above)
Comment (Wynndow) [inline VERSION]: Fair dos

**Pull request merged**